### PR TITLE
docs & github issue template - update bug issue triage

### PR DESCRIPTION
**Pull request by lb-**
This PR aims to provide more clarity about what the bug/issue triage steps should look like, with less emphasis on milestones being the indicator of a bug being triaged. Instead we will add `status:Unconfirmed` to all new bugs and use the removal of this label to communicate that it has been triaged (at which point, the bug may alternatively be closed or converted to a different type like an enhancement).

### Details
- convert `docs/contributing/issue_tracking.md` from RST to markdown
- add details about when to use Github discussions instead of an issue
- add details about bugs will be raised with the [`status:Unconfirmed`](https://github.com/wagtail/wagtail/labels/status%3AUnconfirmed) by default
- add link to PR triage guide
- add links to release schedule & roadmap
- put less emphasis on milestones for new issue creation
- be less explicit about us marking things with an invalid label but rather that the issue will just be closed
- update bug template to add the other label by default and also move some parts into HTML comments that are more like a 'guide' for a new issue (we already do this in other issue templates).

### Further actions if merged in
- Remove the `invalid` label https://github.com/wagtail/wagtail/issues?q=label%3Ainvalid+ (TBC)
- communication on announcements about this change to new bug creation (TBC)

### CODE REVIEW

File: docs/contributing/issue_tracking.md [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+We welcome bug reports, feature requests and pull requests through Wagtail's [Github issue tracker](https://github.com/wagtail/wagtail/issues).
 
-Issues
-------
+## Issues
 
-An issue must always correspond to a specific action with a well-defined completion state: fixing a bug, adding a new feature, updating documentation, cleaning up code. Open-ended issues where the end result is not immediately clear ("come up with a way of doing translations") are fine, as long as there's a clear way to progress the issue and identify when it has been completed (not e.g. "make rich text fields suck less").
+An issue must always correspond to a specific action with a well-defined completion state: fixing a bug, adding a new feature, updating documentation, or cleaning up code. Open-ended issues where the end result is not immediately clear ("come up with a way of doing translations" or "make rich text fields suck less") are better suited to [Github discussions](https://github.com/wagtail/wagtail/discussions), so that there can be feedback on clear way to progress the issue and identify when it has been completed through separate issues created from the discussion.
```
Comment (allcaps) [inline docs/contributing/issue_tracking.md:7]: Can we have more positive example?
`make rich text fields suck less` -> `Add HTML-structures to richt text`. 
Or something else discussion worthy.

**Review by allcaps — APPROVED** [review]
LGTM, only the wording of the example title.

File: docs/contributing/issue_tracking.md (hunk 1 above)
Comment (lb-) [reply docs/contributing/issue_tracking.md:7]: Just saw this, fixing now.

**Review by allcaps — APPROVED** [review]

**Comment by lb-** [comment]
Thanks @allcaps

**Pull request merged**